cinder/volume/rpcapi.py

```diff
@@ -191,7 +191,7 @@ def delete_volume(self, ctxt, volume, unmanage_only=False, cascade=False):
 
         cctxt.cast(ctxt, 'delete_volume', **msg_args)
 
-    def create_snapshot(self, ctxt, volume, snapshot):
+    def create_snapshot(self, ctxt, volume, snapshot, backend=None):
```

Comment:
Same as cinder/volume/manager.py#L1181.
```suggestion
    def create_snapshot(self, ctxt, volume, snapshot, _backend):
```

Reply:
same